templates/registration/registration_closed.html

```diff
@@ -0,0 +1,3 @@
+{% block body %}
+    <p>{{ _('We temporarily closes the user registration. For enquiries, please contact the administrator.') }}</p>
```

Comment:
Can you change the message to `Registration is currently closed. Please contact an administrator.`